tests/test_lab_train.py

```diff
@@ -1,5 +1,5 @@
-# SPDX-License-Identifier: Apache-2.0
 # pylint: disable=R0801
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Why moving the lines?

Reply:
There was a pylint warning/error here, which prevent me from making commit.